spring-cloud-azure-samples/eventhubs-binder-sample/src/main/java/com/example/SourceExample.java

```diff
@@ -36,4 +36,9 @@ public String postMessage(@RequestParam String message) {
     public String hello() {
         return "hello world";
     }
+
+    @GetMapping("/")
```

Comment:
Do we need this function? Seems like it's the same as `/hello`.

Reply:
it's useful when we deploy the event hub on App Service. if the service is running, when we call the App Service Url, it will return welcome and notice us the service is running, otherwise, the page will report `can not find the /"

Reply:
Not sure whether the error was reported by the Spring framework. But let's keep this.